packages/ast-spec/src/ast-node-types.ts

```diff
@@ -88,9 +88,9 @@ export enum AST_NODE_TYPES {
   WhileStatement = 'WhileStatement',
   WithStatement = 'WithStatement',
   YieldExpression = 'YieldExpression',
-  /**
-   * TS-prefixed nodes
-   */
+
+  // TS_prefixed nodes
```

Comment:
The `/** ... */` JSDoc style comment was getting picked up as a comment on specifically `TSAbstractAccessorProperty`.